Tests/UnitTests/Security/SigningTests.swift

```diff
@@ -502,10 +502,100 @@ class SigningTests: TestCase {
         ) == true
     }
 
+    func testVerifyKnownSignatureForGetRequestWithSignedHeaders() throws {
+        // swiftlint:disable line_length
+        /*
+         Signature retrieved with:
+        curl -v 'https://api.revenuecat.com/v1/subscribers/$RCAnonymousID%3A6ca4535c42714f88abc99c563703f113' \
+        -X GET \
+        -H 'X-Nonce: MTIzNDU2Nzg5MGFi' \
+        -H 'X-Is-Sandbox: true' \
+        -H 'X-Headers-Hash: X-Is-Sandbox:sha256:b5bea41b6c623f7c09f1bf24dcae58ebab3c0cdd90ad966bc43a45b44867e12b' \
+        -H 'Authorization: Bearer appl_fFVBVAoYujMZJnepIziGKVjnZBz' \
+         */
+
+        let response = """
+        {"request_date":"2023-12-08T19:17:04Z","request_date_ms":1702063024731,"subscriber":{"entitlements":{},"first_seen":"2023-12-08T19:13:02Z","last_seen":"2023-12-08T19:13:02Z","management_url":null,"non_subscriptions":{},"original_app_user_id":"$RCAnonymousID:6ca4535c42714f88abc99c563703f113","original_application_version":null,"original_purchase_date":null,"other_purchases":{},"subscriptions":{}}}\n
+        """
+        let expectedSignature = "x2qnlHOl5WuzGi4TbSUVHxzlKELRCfrRYG9XAiso7ucZTQAAYEZqbguA3X0YfCJqCKh2hnTLSdEr4R+t23xBlTxceWZu2TJjK3461UJKpUnrwXDv+tYo2K54IoS3/tsEr3VmB5ppKAq0P2CR7SwbsDPpxUlHBcl5/4XJvb/DHOnTKjIVd4WJ+57LLWvIV9sDHnj9XxiBez+p5cEjez1RtUis0XdCfAFXU8XfAq6ggiEJKX4F"
```

Comment:
I've generated these from the backend, so we can confirm that these real signatures are valid.